src/main/java/org/javarosa/xpath/expr/XPathJoinChunkFunc.java

```diff
@@ -0,0 +1,66 @@
+package org.javarosa.xpath.expr;
+
+import org.javarosa.core.model.condition.EvaluationContext;
+import org.javarosa.core.model.instance.DataInstance;
+import org.javarosa.xpath.XPathArityException;
+import org.javarosa.xpath.XPathNodeset;
+import org.javarosa.xpath.parser.XPathSyntaxException;
+
+public class XPathJoinChunkFunc extends XPathFuncExpr {
```

Comment:
We don't appear to document XPath functions that way anymore, I updated the json doc that appears to be where these docs go now.